infor-design/enterprise-ng
# 1186 - Added example page to test category update

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added an example page to test if searchfield category popupwrapper is not creating duplicate elements on update.

Requires this PR to work: https://github.com/infor-design/enterprise/pull/6230

**Related github/jira issue (required)**:

Closes #1186 

**Steps necessary to review your pull request (required)**:

- Update IDS to latest version
- Pull this branch, build and run the app
- Go to: http://localhost:4200/ids-enterprise-ng-demo/searchfield-category-update
- Inspect the page
- Click on Update button several times
- There should be no new popup wrapper elements created
